docs/contributing/adding-icons-to-kicon.md

```diff
@@ -0,0 +1,38 @@
+# Adding a new icon to KIcon
+There are a number of steps required to add a new icon file to [KIcon](/components/icon.html). Follow these steps to ensure the component can properly render the SVG file.
+
+1) **Adding the new icon**  
+    - To start, copy your svg source code into [SVGOMG](https://jakearchibald.github.io/svgomg/) to compress and clean up. You should be fine using the default settings however, feel free to play around to get the most optimal output.
+    :::tip 
+    You can copy the svg by exporting it directly from Figma or by opening the file that design sends over.
+    :::
+    - Once you have run the svg through SVGOMG, copy it out and paste as a new file in the `/packages/KIcon/icons/` folder with the name `icn-{iconName}.svg`
+
+1) **Add to KIcon**  
+  Now that the file has been saved to the `icons` folder import and export it in `/packages/KIcon/icons/index.js`
+    ```js
+    ...
+    import {iconName} from './icn-{iconName}.svg'
+    ...
+
+    export default {
+      ...
+      {iconName}
+      ...
+    }
+    ```
+
+1) **Update Tests**  
+The last step is to ensure the `KIcon` tests pass. Each icon is tested to ensure it matches an existing snapshot. Because this is a new icon you will need to update the snapshots. From The root directory run `yarn test packages/KIcon/KIcon.spec.js --updateSnapshot`.
+
+Thats it you're done! Git push and PR! :tada:
+
+::: warning
+KIcon can only contain path elements. If the icon you are bringing in includes circle, ellipse, rect, text, etc those elements will not be displayed. It is recommended to update your icon to only paths before bringing into Kongponents.
+:::
+
+<!-- Add file to KIcon/icons with name format icn-<icon_name>.svg
```

Comment:
Did you maybe want to mention how to update/view the docs to verify that the icon is rendering correctly? Or is that too obvious?

Reply:
I think it's not needed as we have docs earlier on explaining how to start the app.